report/sections/TestRunner/chapter.tex

```diff
@@ -0,0 +1,33 @@
+\chapter{Test Runner} \label{chap:TestRunner}
+In chapter \ref{chap:Frontend}, we described how running multiple GHC interpreter processes asynchronously eventually crashes the Test Runner web server given sufficiently many simultaneous requests.
```

Comment:
Skrev vi om det i Frontend?

Reply:
Can you fix the ref to point to test runner sec? Or @Pattrigue / @Ivikhostrup

Reply:
Done